source/documentation/deploying_services/use_a_custom_domain.md

```diff
@@ -15,79 +17,283 @@ You must configure the `cdn-route` service to use a subdomain. If you configure
+
+You have now completed the custom domain setup process, and can now [map a subdomain to your app](#mapping-a-subdomain-route-to-your-app).
+
+It should take one hour for domain setup to finish after you have created the required DNS records. If it has not finished after 2 hours, please refer to the [troubleshooting](/deploying_services/use_a_custom_domain/#troubleshooting-custom-domains) section.
+
+You must [remove a custom domain from your cdn-route service](#removing-a-custom-domain-from-your-cdn-route-service) to stop using any of your custom domains with the GOV.UK PaaS.
+
+You can associate up to 100 subdomains with a single cdn-route service instance.
+
+### Mapping a subdomain route to your app
+
+For each subdomain on which you want your app to be accessible, map the route to the app:
+
+```
+cf map-route APP_NAME --hostname www DOMAIN_NAME
 ```
 
-The maximum number of domains that can be associated with a single cdn-route service instance is 100.
+Your app is now ready to receive requests through the subdomain.
+
+### Removing a custom domain from your cdn-route service
+
+<div class="govuk-warning-text">
+  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+  <strong class="govuk-warning-text__text">
+    <span class="govuk-warning-text__assistive">Warning</span>
+    <p>Removing the domain results in the service provided by the app on that domain becoming unavailable.</p>
+  </strong>
+</div>
+
+1. Your org manager should check if any apps are using a custom domain before removing that domain by running:
+
+    ```
+    cf routes --orglevel | grep -we DOMAIN_NAME -e^space
+    ```
+    In this example, your org manager must tell the owners of `app-1` and `app-2` that you are removing the domain:
+
+    ```
+    space   host     domain        port   path   type   apps     service
+    tools   www      DOMAIN_NAME                        app-1
+    tools   info     DOMAIN_NAME                        app-2
+    ```
+
+
+1. Remove the custom domain from Cloud Foundry's routing database and from every app using that domain:
+
+    ```
+    cf delete-domain DOMAIN_NAME
+    ```
+
+1. Check how many custom domains your cdn-route service has:
+
+    ```
+    cf service SERVICE_INSTANCE | grep ^message:
+    ```
+
+    where `SERVICE_INSTANCE` is the name of one of your existing cdn-route service instances.
+
+    In this example, there are 2 configured domains contained inside the square brackets: `www.example.com` and `www.example.net`.
+
+    ```
+    message: Service instance provisioned [www.example.com,www.example.net =>
+             london.cloudapps.digital]; CDN domain d3nrs0916m1mk2.cloudfront.net
+    ```
+
+    If you cannot see this output, please contact GOV.UK PaaS support at [gov-uk-paas-support@digital.cabinet-office.gov.uk.](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk.) before proceeding.
```

Comment:
You have a spurious trailing `.` in the `mailto` link. This is also why Travis is failing.

Reply:
changed